src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -229,20 +244,22 @@ def __stringToByte(self, string):
         """
         Convert one "string" byte (like '0xFF') to real integer byte (0xFF).
 
-        @param string string
-        @return byte
+        Arguments:
+            string (str): The string
+
+        Returns:
+            The byte (int)
         """
 
         return struct.unpack('@B', string)[0]
 
     def __writePacket(self, packetType, packetPayload):
         """
-        Send a packet to fingerprint sensor.
-
-        @param integer(1 byte) packetType
-        @param tuple packetPayload
+        Sends a packet to the sensor.
 
-        @return void
+        Arguments:
+            packetType (int): The packet type (either FINGERPRINT_COMMANDPACKET, FINGERPRINT_DATAPACKET or FINGERPRINT_ENDDATAPACKET)
```

Comment:
Write inline code in markdown style with ticks ``, e.g. `FINGERPRINT_COMMANDPACKET`